### DISCUSSION TITLE
Add option for listed up to 2 months ago please

### DISCUSSION THREAD

**Discussion by MartinAntoniuk** [SnosMe/awakened-poe-trade#689] [Ideas]
Hi, I play standard and I would love to be able to search for offline items but that aren't up for years now.
This is specially important for synthesied and fractured items (but would be appreciated for all items), which there aren't many online on the market, but that are flooded of 2 year old offline items from synthesis league or people that don't play anymore.

If we could have a hidden option somewhere were we can choose to do offline searches with "up to" time, it would be amazing, really time saving and I would appreciate it very much.

Thank you very much for this amazing tool and for all your time and energy. <3

**Comment by SnosMe** [comment]
I agree that this is a reasonable default value, and it will be activated when you do an offline search, so 1 less click for you.

P.S. At the beginning of Kalandra league, trade site was fully reindexed, and because we're only 1.5months into league it may look like the filter doesn't work.

![image](https://user-images.githubusercontent.com/4292308/194717736-ba2bc3ff-ffae-4cee-ad26-39434d8af56d.png)

**Reply by MartinAntoniuk** [reply]
Thank you so much <3 This will be immensely useful in the future.
